NIPTool/server/views.py

```diff
@@ -342,3 +354,34 @@ def update():
                 )
 
     return redirect(request.referrer)
+
+
+@server_bp.route("/load", methods=["POST"])
+def load():
```

Comment:
Is this function invoked by a POST request from a web page of the same app or also some external program? It's a bit hard for me to understand how this works..

Reply:
Its not supposed to be used from a web page, but only from external proiograms such as postman or cg. I should put the two load functiuons in a separete blueprint maybe? For clarification

Reply:
Ok. I've asked because `return redirect(request.referrer)` is used within the app to redirect to the previous webpage, where the request to this endpoint was generated. You should remove it then and return a response instead